.releaserc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  branches: ['release'],
+  plugins: [
+    '@semantic-release/commit-analyzer',
+    '@semantic-release/release-notes-generator',
+    [
+      '@semantic-release/github',
+      {
+        assets: [
+          { path: 'CHANGELOG.md' },
+          { path: 'LICENSE' },
+          { path: 'README.md' },
+        ],
+      },
+    ],
+    [
+      '@semantic-release/git',
+      {
+        assets: ['CHANGELOG.md', 'package.json', 'projects/**/package.json'],
```

Comment:
I wonder if `package-lock.json` must be included too (version bump).

Reply:
I have no idea

Reply:
I think I'll add it.. doesn't matter really if it isn't needed.